### PR TITLE
[dev] Format JSON and Markdown better

### PR DESCRIPTION
**Pull request by niedzielski**
Wikimediaify the JSON and Markdown files.

Bug: T256693

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -4,34 +4,35 @@ Versions and bullets are arranged chronologically from latest to oldest.
 
 ## v0.0.1 (unreleased)
```
Comment (Volker-E) [inline changelog.md:6]: Without looking at this with a Markdown viewer (not sure if GitHub diffing is showing me everything), doesn't this change the rendering?  
In OOUI  we leave them unintended

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Volker-E) [inline .prettierrc.json:1]: Out of curiosity. Wouldn't it be better to use an eslint plugin for this?

Comment (niedzielski) [reply .prettierrc.json:1]: In my experience, Prettier's ability to apply automatic formatting is far superior to ESLint (all formatting changes in this patch were applied automatically without exception, for example). It also supports many languages without custom community maintained plugins of varying quality (eslint-plugin-json is used in this repo, for example, but it supports very few options). Finally, Prettier is ESLint compatible.

If you haven't tried it out yet, please do! [The Prettier playground](https://prettier.io/playground/) is a great place to start. I've been using it for a couple years and haven't seen anything quite like it. In a nutshell, Prettier is unbelievably good at accepting utter garbage code in and generating beautiful and extremely consistent outputs. Its changed the way I read and write code for repositories where its used.

File: changelog.md (hunk 1 above)
Comment (niedzielski) [reply changelog.md:6]: This is tab width of four. I see no difference locally or in GitHub's rich diff.

diff

File: .prettierrc.json (hunk 2 above)
Comment (speedandfunction-anribolon) [reply .prettierrc.json:1]: In addition, Prettier is compatible with the most of [editors/IDEs](https://prettier.io/docs/en/editors.html)

Comment (Volker-E) [reply .prettierrc.json:1]: @speedandfunction-anri Interesting, which IDEs are incompatible with eslint?

**Pull request merged**